scripts/automation/Radius/Generate-Cert.ps1

```diff
@@ -0,0 +1,46 @@
+# this script will generate a Self Signed CA (root cert) to be imported on the
+# Radius CBA-BYO Authentication UI
+
+# Edit the variables in Config.ps1 before running this script
+. "$psscriptRoot/Config.ps1"
+
+if ( ([System.String]::IsNullOrEmpty($JCORGID)) -Or ($JCORGID.Length -ne 24) ) {
+    throw "OrganizationID not specified, please update config.ps1"
+}
+
+################################################################################
+# Do Not Edit Below:
+################################################################################
+cd $PSSCRIPTROOT
+$opensslBinary = '/usr/local/Cellar/openssl@3/3.0.7/bin/openssl'
```

Comment:
Couldn't we import the declared variable from the config.ps1 rather than redeclaring?

Reply:
Yep let's remove this here :) I'll do that shortly